README.md

```diff
@@ -1,5 +1,19 @@
+[![Bonsai Asset Badge](https://img.shields.io/badge/Sensu%20Prometheus%20Collector-Download%20Me-brightgreen.svg?colorB=89C967&logo=sensu)](https://bonsai.sensu.io/assets/sensu/sensu-prometheus-collector) [![Build Status](https://travis-ci.org/sensu/sensu-prometheus-collector.svg?branch=master)](https://travis-ci.org/sensu/sensu-prometheus-collector)
 ## Sensu Prometheus Collector
 
+- [Overview](#overview)
+- [Usage examples](#usage-examples)
+- [Configuration](#configuration)
+  - [Sensu Go](#sensu-go)
+    - [Asset manifest](#asset-manifest)
```

Comment:
```suggestion
    - [Asset definition](#asset-definition)
```

Reply:
These anchor links may need some work, to make them distinct from their Sensu Core counterparts